pymatgen/analysis/ferroelectricity/ZVAL.yaml

```diff
@@ -0,0 +1,305 @@
+VASP_PBE_ZVAL:
```

Comment:
Is there a need for this file. This should be available from the POTCAR itself? In general, I am not a fan of hard coding these because you will need to maintain these with changes in VASP PSP.